pkg/secrets/clients/k8s/k8s_secrets_client.go

```diff
@@ -32,15 +28,21 @@ func PatchK8sSecret(namespace string, secretName string, stringDataEntriesMap ma
 	// get K8s client object
 	kubeClient, _ := configK8sClient()
 
-	stringDataEntry, err := generateStringDataEntry(stringDataEntriesMap)
+	log.Info(messages.CSPFK006I, secretName, namespace)
```

Comment:
The message in `CSPFK006I ` is `Patching Kubernetes secret '%s' in namespace '%s'. It seems that it is misplaced here as you are getting the secret and not patching it.

Also, please change the message to say that you are updating the secret, and not that you are patching it (as you changed the REST method).

Reply:
done